### ISSUE TITLE
accessibility-checker-test-basic randomly fails

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#90]
Git action test accessibility-checker-test-basic sporadically fails:

```
1) "before all" hook in "{root}":
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
      at listOnTimeout (internal/timers.js:549:17)
      at processTimers (internal/timers.js:492:7)
```

I have not yet hunted down what "before all" hook it's talking about. Works locally, and when adding in console statements, all of the before statements in the tests seem to complete.

**Comment by tombrunet** [comment]
This appears to be fixed now that the tests have been split into individual runs, and found one before error. Poor error reporting by mocha.
